<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atomMainWin</class>
 <widget class="QMainWindow" name="atomMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>696</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>846</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ATOM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progress">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseResumeBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showDetailsCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openReportBtn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>610</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Report</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>821</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#000000&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newProjectBtn"/>
    <addaction name="openProjectBtn"/>
    <addaction name="saveBtn"/>
    <addaction name="saveAsBtn"/>
    <addaction name="exitBtn"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="docBtn"/>
    <addaction name="aboutBtn"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="settingsBtn"/>
    <addaction name="runBtn"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newProjectBtn">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="openProjectBtn">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="saveBtn">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="docBtn">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="aboutBtn">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="saveAsBtn">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="exitBtn">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="settingsBtn">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="runBtn">
   <property name="text">
    <string>Run Experiment</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
